packages/malloy/src/lang/test/parameters.spec.ts

```diff
@@ -696,4 +696,38 @@ describe('parameters', () => {
       run: foo_ext -> { select: param_value }
     `).toTranslate();
   });
+  test('can declare and use a filter expression type as a parameter type', () => {
+    expect(`
+      ##! experimental.parameters
+      source: ab_new(goodNumbers::filter<number>) is ab extend { where: ai ~ goodNumbers }
+      source: single_digits is ab_new(goodNumbers is f'>=0 and <= 9')
+    `).toTranslate();
+  });
+  test('filter expression parameters must have types with default values', () => {
+    expect(`
+      ##! experimental.parameters
+      source: ab_new(goodNumbers is f'[25 to 50]') is ab
+    `).toLog(
+      errorMessage(
+        "Filter expression parameters must have expicit filter type, for example 'goodNumbers::filter<string>'"
```

Comment:
Would also like chris to agree to this

Reply:
Agree. Though as noted above, I would prefer if the use could choose to put the cast after the filter, as you can with null-valued defaults.